README.md

```diff
@@ -1,5 +1,5 @@
 [![pipeline status](https://gitlab.com/math-comp/math-comp/badges/master/pipeline.svg)](https://gitlab.com/math-comp/math-comp/commits/master)
```

Comment:
I thought this one has also to be replaced with Nix CI.

Reply:
Let's merge this first